src/services/subscriptions/subscription-service/subscription.publish.service.ts

```diff
@@ -47,11 +48,13 @@ export class SubscriptionPublishService {
     roomId: string,
     type: MutationType,
     data: IMessage | IMessageReaction,
-    messageID?: string
+    messageID?: string,
+    room?: Room
   ): Promise<void> {
     const payload: RoomEventSubscriptionPayload = {
       eventID: `room-event-${randomInt()}`,
       roomID: roomId,
+      room: room || {},
```

Comment:
I am not convinsed this won't throw if it's an empty object. Why is `room` optional to begin with - it's mendatory in `RoomEventSubscriptionResult`